core-web/libs/block-editor/src/lib/shared/plugins/floating.plugin.ts

```diff
@@ -86,6 +86,8 @@ export class FloatingActionsView {
         this.command = command;
         this.key = key;
         this.tippyOptions = tippyOptions;
+
+        document.addEventListener('DOMContentLoaded', () => this.setButtonPosition());
```

Comment:
Here you are referencing an anon arrow function, while in the remove you reference directly `this.setButtonPosition`, you need to make sure you reference both the same so the `removeEventListener` works.

```suggestion
        document.addEventListener('DOMContentLoaded', this.setButtonPosition);
```